docs/UsersGuide/source/BackgroundInfo/Introduction.rst

```diff
@@ -8,30 +8,41 @@ The Unified Forecast System (:term:`UFS`) is a community-based, coupled, compreh
 
 The UFS includes `multiple applications <https://ufscommunity.org/science/aboutapps/>`__ that support different forecast durations and spatial domains. This documentation describes the UFS Short-Range Weather (SRW) Application, which targets predictions of atmospheric behavior on a limited spatial domain and on time scales from minutes to several days. The most recent SRW Application includes a prognostic atmospheric model, pre- and post-processing, and a community workflow for running the system end-to-end. These components are documented within this User's Guide and supported through the `GitHub Discussions <https://github.com/ufs-community/ufs-srweather-app/discussions/categories/q-a>`__ forum. The SRW App also includes support for a verification package (METplus) for both deterministic and ensemble simulations and support for four stochastically perturbed physics schemes. 
 
-Since the v2.1.0 release, developers have added a variety of features:
+Since the last release, developers have added a variety of features:
 
    * Bug fixes since the v2.1.0 release
-   * Pre-implementation Rapid Refresh Forecast System (RRFS) forecast configurations
-   * Air Quality Modeling (AQM) capabilities
-   * Updates to :term:`CCPP` that target the top of the ``main`` branch (which is ahead of CCPP v6.0.0). See :ref:`this page <CCPPUpdates>` for a detailed summary of updates that came in ahead of the v2.1.0 release.
-   * Support for the :term:`UPP` inline post option (see :ref:`here <InlinePost>`)
-   * Documentation updates to reflect the changes above
-
-The SRW App v2.1.0 citation is as follows and should be used when presenting results based on research conducted with the App:
-
-UFS Development Team. (2022, Nov. 17). Unified Forecast System (UFS) Short-Range Weather (SRW) Application (Version v2.1.0). Zenodo. https://doi.org/10.5281/zenodo.7277602
+   * Addition of the supported ``FV3_RAP`` physics suite (`PR #811 <https://github.com/ufs-community/ufs-srweather-app/pull/811>`__) and support for the ``RRFS_NA_13km`` predefined grid
```

Comment:
Is `FV3_GFS_v17_p8` also considered fully supported? If so, which grids does it play nice with? (I'd have to document that) 
@mkavulich feel free to chime in ;)

Reply:
I think what you have done in hash [0d02479](https://github.com/ufs-community/ufs-srweather-app/pull/950/commits/0d02479f116665dde3dd91b44f66e9cb32f6ca2c) is sufficient.  This physics suite was added because it was being used in the weather model, but not in the SRW App.  However, I'd let others make the final decision on whether this physics suite is fully supported, or just added to ensure that we are up to date with respect to the physics suites that are used in the weather model.

Reply:
I will go ahead and resolve this conversation now.